DTR_futbol.md

```diff
@@ -0,0 +1,77 @@
+Project: Futbol
+
+Group Member Names: Jake Heft, Joshua Carey, Hope Gochnour, Will Dunlap
+
+Goals and Expectations for the Project (What does each group member hope to get out of this project? What do we want to achieve as a team? How will we know that we're successful?):
+
+- Jake: Get better at planning a project (grow in project organization skills) and refine GH workflow.
+- Hope: Get better at handling how files are read from and how to work in a group.
+- Joshua: Get better at approaching project with big picture in mind.
+- Will: Learn effective ways of how to break up the project that saves time and makes coding more efficient. Growing in articulating thoughts/ideas before execution.
+- Project Driver: getting better and learning new stuff.
+
+Team strengths & collaboration styles (consider discussing your Pairin qualities here):
+
+- Jake: Would like to see a mix of both individual and collaborative coding.
+- Joshua: Preference towards independent work in terms of creativity.
+- Will: Most efficient as independent worker and with intermittent breaks.
+- Hope: Preference for mix of independent and direct collaboration. As needed, pair off or
+- work together as a group. Outside of call check-ins, independent or pair up if needed.
+
+<!-- How we can use our strengths to overcome obstacles: -->
+Schedule Expectations (When are we available to work together and individually? What constraints do we have? Make sure to discuss the remote questions above in regards to this section):
+
+- Hope: Friday night commitments and Sunday night commitments.
+- Jake: 1st and 2nd Saturday commitments. Ideally finish around 8 and 9, but week is open.
+- Joshua: 1st weekend gone and open 2nd except Sundays 4 to 8 pm.
+- Will: Flexible weekend commitments. Tuesdays (4-6 commitment but flexible)
+- During weeK: Stop around 9 or 10 pm each night.
+- Keep 5 to 7pm open for a zoom-like study hall.
+- Have at least one zoom check-in per day (usually during )
+
+Communication Expectations (How and often will we communicate? How do we keep lines of communication open? How will we make decisions as a team? How will we communicate about our emotional and mental well-being with each other?):
+
+- Zoom:
+- Slack: open communication through slack channel
+- GoogleDoc: Hope will start a GoogleDoc for project question organization
+- GitHub: comments on PRs and responses  
+
+  *Make sure everyone is tracking; no dumb questions*
+  *Get in the habit of providing and receiving feedback in GH comments*
+  *In check-ins, have a quick touch point about how communication and project is going.*
```

Comment:
Also just a check in to see how everyone is feeling, if anything's come up in life that would be helpful for team members to be aware of.